control/src/test/java/org/kaazing/k3po/control/internal/ControlTest.java

```diff
@@ -127,6 +127,7 @@ public void shouldWritePrepareCommand() throws Exception {
         final byte[] expectedPrepare =
                 ("PREPARE\n" +
                  "version:2.0\n" +
+                 "content-length:0\n" +
```

Comment:
Is this a backward compatible change to the protocol? Maybe bump protocol version?